data/json/vitrification/terrain_vitrified.json

```
@@ -0,0 +1,603 @@
+    "description": "A spar of glass fused into the ground, still bearing the jagged edges of the structure it once supported..",
+    "symbol": "#",
+    "color": "dark_gray",
+    "looks_like": "t_fence_post",
+    "move_cost": 2,
+    "flags": [ "TRANSPARENT", "THIN_OBSTACLE", "SHARP" ],
+    "bash": {
+      "str_min": 8,
+      "str_max": 20,
+      "sound": "crack.",
+      "sound_fail": "whump.",
+      "ter_set": "t_vitrified_sand",
+      "items": [ { "item": "black_glass_shard", "count": [ 40, 80 ] } ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_vitrified_water_pump",
+    "name": "black glass well",
+    "looks_like": "f_machinery_heavy",
+    "description": "This was formerly a well shaft and pump mechanism; it is now so obscured with bubbles of black glass that only a faint outline of the original shape can be seen..",
+    "symbol": "&",
+    "color": "light_gray",
+    "move_cost": 6,
+    "coverage": 40,
+    "flags": [ "TRANSPARENT", "THIN_OBSTACLE", "SHARP" ]
+  },
+  {
+    "type": "terrain",
+    "id": "t_vitrified_bars",
+    "name": "black glass bars",
+    "description": "Thick spars of black glass extending from floor to ceiling..",
+    "examine_action": "bars",
+    "symbol": "\"",
+    "color": "light_gray",
+    "move_cost": 0,
+    "flags": [ "TRANSPARENT", "NOITEM", "PERMEABLE", "CONNECT_WITH_WALL", "THIN_OBSTACLE" ],
+    "bash": {
+      "str_min": 60,
+      "str_max": 250,
+      "sound": "a tremendous shattering!",
+      "sound_fail": "clonk!",
+      "ter_set": "t_vitrified_sand",
+      "items": [ { "item": "black_glass_shard", "count": [ 40, 80 ] } ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_vitrified_column",
+    "name": "black glass column",
+    "looks_like": "t_nether_glass_wall",
+    "description": "A tremendous spar of black glass.",
+    "symbol": "1",
+    "color": "light_gray",
+    "move_cost": 0,
+    "coverage": 80,
+    "flags": [ "WALL", "PERMEABLE", "MINEABLE" ],
+    "bash": {
+      "str_min": 120,
+      "str_max": 200,
+      "sound": "crash!",
+      "sound_fail": "whump!",
+      "ter_set": "t_vitrified_sand",
+      "items": [ { "item": "black_glass_shard", "count": [ 40, 80 ] } ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_vitrified_pool",
+    "name": "shallow black pool",
+    "description": "At first glance, this looks like a smooth pool of black water. Closer examination reveals it to be fine glass dust.",
+    "symbol": "~",
+    "looks_like": "t_water_pool",
+    "color": "black",
+    "move_cost": 7,
+    "flags": [ "TRANSPARENT", "LIQUID", "NO_SCENT", "SWIMMABLE", "SHARP" ],
+    "connect_groups": "POOLWATER",
+    "connects_to": "POOLWATER"
+  },
+  {
+    "type": "terrain",
+    "id": "t_tree_apple_vitrified",
+    "name": "tree of glass apples",
+    "looks_like": "t_tree",
+    "description": "A flourishing tree, its branches heavy with apples. Every black leaf is razor-edged; every heavy fruit turns to dust at the touch.",
```

```suggestion
    "description": "A flourishing tree, its branches heavy with apples.  Every black leaf is razor-edged; every heavy fruit turns to dust at the touch.",
```
~~How does the player know the fruits turn to dust on touch when they're only looking at it~~